### PR TITLE
Type of sensors

### LINKED ISSUE #12: type of sensors
Currently, the base class of a sensor has no information, apart from the name. It might be advantageous to store information related to what is measured (e.g. displacements, temperature..). Even though this could be added to the name, it is easier to loop in the __call__ function of the forward model over the sensors and then decide based on the type what is returned.

### PR DESCRIPTION
**Pull request by aklawonn**
This fixes #12

### CODE REVIEW

File: probeye/definition/sensor.py [hunk 1]
```diff
@@ -12,27 +12,39 @@ class Sensor:
...
+        self.unit = unit
 
 
 class PositionSensor(Sensor):
     """
     Class for a forward model's sensor with constant positional coordinates.
     """
-    def __init__(self, name, x=None, y=None, z=None, coords=None,
-                 order=('x', 'y', 'z')):
+    def __init__(self, name, measurand='position', unit='', x=None, y=None,
```
[the remark this answers is not in the document]
Comment (aklawonn) [reply probeye/definition/sensor.py]: Changed it. However, maybe we should think about if the naming of this sensor is ideal. When something is called TemperatureSensor, I would think that it measures the temperature. I wouldn't blame anybody who thinks the same about a PositionSensor, i.e., that such a sensor measures the position. One alternative would be to rename it to "PositionedSensor". What do you think?

[the remark this answers is not in the document]
Comment (joergfunger) [reply probeye/definition/sensor.py]: You are right that position sensor is maybe biased. What about SensorWithCoordinates? Or,maybe even more intuitive, we provide for the base class an additional argument coordinates?

[the remark this answers is not in the document]
Comment (aklawonn) [reply probeye/definition/sensor.py]: All right, I changed it to SensorWithCoordinates. I like to have a simple Sensor base class, without the functionality for coordinates.

**Comment by aklawonn** [comment]
As discussed on the phone I combined PositionSensor and Sensor to a new basic Sensor class.

**Pull request merged**